timescale-container/test/ql-db-setup.sh

```diff
@@ -0,0 +1,45 @@
+#!/usr/bin/env bash
```

Comment:
i am not sure i would refer to lusovini in a public repo.

Reply:
got rid of it, thx!